models/shopify__discounts.sql

```diff
@@ -1,8 +1,7 @@
 {{
     config(
-        materialized='table' if target.type in ('bigquery', 'databricks', 'spark') else 'incremental',
+        materialized='table',
```

Comment:
Do we need any of this configuration anymore now that the incremental strategy is being removed? Why do we need to define the cluster and unique_key if there is no incremental strategy leveraging them?

Reply:
Same question on the other incremental updates in this PR.

Reply:
Agreed. Removed.